index.html

```diff
@@ -2346,9 +2346,9 @@ <h3>Global State</h3>
      <li><p>If the list of <a>active sessions</a> is empty, set
      the <a>webdriver-active flag</a> to false.
 
-     <li><p>An <a>endpoint node</a> must <a>close</a> any <a>top-level
-      browsing contexts</a> associated with the <a>session</a>,
-      without <a>prompting to unload</a>.
+     <li><p>Optionally, <a>close</a> any <a>top-level browsing contexts</a>
```

Comment:
Should we clarify what `optionally` means? Is it the case when there are still entries listed in `sessions`?